win32/Lib/win32gui_struct.py

```diff
@@ -994,6 +994,8 @@ def UnpackDEV_BROADCAST(lparam):
         _, _, _, x["unitmask"], x["flags"] = struct.unpack(
             fmt, buf[: struct.calcsize(fmt)]
         )
+    elif devtype == win32con.DBT_DEVTYP_PORT:
```

Comment:
Should this also have `devicetype`?

Reply:
Yes, this structure is returned if a port is connected, like COM or LPT. Contains the standard header, including devicetype and port name (string).

typedef struct _DEV_BROADCAST_PORT_W {
  DWORD   dbcp_size;
  DWORD   dbcp_devicetype;
  DWORD   dbcp_reserved;
  wchar_t dbcp_name[1];
} DEV_BROADCAST_PORT_W, *PDEV_BROADCAST_PORT_W;